styles/main.scss

```diff
@@ -53,6 +53,9 @@ hr {
   height: 1px;
   border: none;
   background: #e8e8e8;
+  position: absolute;
+  width: 100%;
+  left: 0;
 }
```

Comment:
Need to check this, I add it here like global style, but it can be like inline style 🤔
In project I didn't find <hr> element